packages/extension-api/src/extension-api.d.ts

```diff
@@ -4367,10 +4373,13 @@ declare module '@podman-desktop/api' {
     };
 
     updateVersion(version: CliToolUpdateOptions): void;
-    onDidUpdateVersion: Event<string>;
+    onDidUpdateVersion: Event<string | undefined>;
```

Comment:
in which case we don't have the version when it's being updated ?

Reply:
when we uninstall. At first i wanted to add the uninstall in this PR as well then i changed my mind as I thought it would have been easier to split them. 
I can remove this part and re-add in the uninstall PR, if preferred.

Reply:
I think it's better to clean it, i'll update the PR

Reply:
maybe yes remove it as it'll be clearer

Reply:
removed 👍